cypress/e2e/shared/scriptQueryBuilder.flux.test.ts

```diff
@@ -93,32 +76,41 @@ describe('Script Builder', () => {
       `fn: (r) => r._measurement == "${measurement}"`
     )
     cy.getByTestID('flux-editor').within(() => {
-      cy.get('.composition-sync--on').should('have.length', 3) // three lines
+      cy.get('.composition-sync--on').should('have.length.gte', 3) // three lines
```

Comment:
Control whether or not the formatting inserts new lines. e.g.
```
|> filter(
       fn: (r) => r._measurement == "${measurement}"
)
```


This is done by:
1. only looking for the measurement itself (`fn: (r) => r._measurement == "${measurement}"`)
2. looking for at least 3 lines added in the sync styling.